docs/source/install/on_complete.rst

```diff
@@ -1,20 +1,15 @@
-|st2| package-based installations have authentication enabled by default, therefore in the end of
-installation you will be provided with the default username and password.
-
-In order to use |st2|, including the commands below, you need to include a token, generated for the
-user, as a CLI argument or be provided as an environment variable.
-
-.. include:: ../auth_usage.rst
-
-Commands below assume that the token was either specified in the :ref:`CLI
-config file <cli-configuration>` or provided as an environment variable.
-
 Check that |st2| installation works OK: ::
 
     st2ctl status
 
     st2 --version
     st2 --help
+
+    # If AUTH enabled: authenticate and export the token env variable
```

Comment:
Latest version (0.11.0) now also supports CLI config so we should tell users to store the credentials in the config instead (docs/source/cli.rst) if they don't want to re-authenticate on every request.